### ISSUE TITLE
What happened to TemporaryDrawer?

### ISSUE THREAD

**Issue by dandv** [prateekbh/preact-material-components#1148]
Seems gone (had an old app generated with preact-cli) and it's also gone from https://material.preactjs.com/component/drawer/. (BTW that page has a syntax error, `drawerOpened = false`).

Nothing in the Changelog.

**Comment by prateekbh** [comment]
yep, it was changed by `mdc-web`. can you pin your version to specific version?
we are addressing these in pmc2.0
